chapter-10-workload-identities/base/authorizationserver/resources/helm-values-template.yaml

```diff
@@ -12,6 +12,9 @@ curity:
       spire-managed-identity: 'true'
     podAnnotations:
       inject.istio.io/templates: 'sidecar,spire'
+    service:
+      annotations:
+        ingress.kubernetes.io/service-upstream: 'true'
```

Comment:
Kong requires this to call upstream components with mTLS.
It forces Kong to connect to the cluster IP rather than individual workload IPs.